### PR TITLE
Feature/sass lint

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

**Comment by vergilius** [comment]
:ok:

**Comment by kdzwinel** [comment]
OK

**Comment by kdzwinel** [comment]
Update readme to link to the correct linter

**Comment by aju** [comment]
Small updates
mixins-before-declarations - added whitelist of mixins
no-mergeable-selectors -disabled - it has issues with selectors added from mixin

mixins-before-declarations - we are using mixin for f

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ This project is a collection of settings for frontend tools used in Brainly
 - .editorconfig - settings for editors [editorconfig.org](http://editorconfig.org/)
 - .eslintrc - config for [eslint.org](http://eslint.org/) with ES2015 enabled
 - .eslintrc-es5 - config for [eslint.org](http://eslint.org/) with only ES5
-- .scss-lint.yml - config for [scss linter](https://github.com/brigade/scss-lint)
+- .ssss-lint.yml - config for [scss linter](https://github.com/sasstools/sass-lint)
```
Comment (kdzwinel) [inline README.md]: :snake:  lol

**Comment by aju** [comment]
Final decision as sass-lint is missing still a lot of features (skipping rules in files, ide support, some rules a bid "dumb") we will stick to scss-lint for now but will keep .sass-lint.yml and hope to use it in future

**Comment by vergilius** [comment]
makes sense :+1:

**Comment by kdzwinel** [comment]
OK

**Pull request merged**